packages/components/src/mobile/bottom-sheet/cell.native.js

```diff
@@ -196,22 +204,30 @@ class BottomSheetCell extends Component {
 					accessibilityHint
 				}
 				onPress={ onCellPress }
-				style={ { ...styles.clipToBounds, ...style } }
+				style={ [ styles.clipToBounds, style ] }
 			>
 				{ drawTopSeparator && (
 					<View style={ separatorStyle() } />
 				) }
-				<View style={ styles.cellContainer } pointerEvents={ containerPointerEvents }>
-					<View style={ styles.cellRowContainer }>
-						{ icon && (
-							<View style={ styles.cellRowContainer }>
-								<Dashicon icon={ icon } size={ 24 } color={ iconStyle.color } />
-								<View style={ platformStyles.labelIconSeparator } />
-							</View>
-						) }
-						<Text numberOfLines={ 1 } style={ [ defaultLabelStyle, labelStyle ] }>
-							{ label }
-						</Text>
+				<View style={ cellContainerStyles } pointerEvents={ containerPointerEvents }>
+					<View style={ rowContainerStyles }>
+						<View style={ styles.cellRowContainer }>
+							{ icon && (
+								<View style={ styles.cellRowContainer }>
+									<Dashicon icon={ icon } size={ 24 } color={ iconStyle.color } />
+									<View style={ platformStyles.labelIconSeparator } />
+								</View>
+							) }
+							<Text numberOfLines={ 1 } style={ [ defaultLabelStyle, labelStyle ] }>
+								{ label }
+							</Text>
+						</View>
+						{ allowReset && <TouchableOpacity onPress={ allowReset } accessibilityRole={ 'button' }>
+							<Text style={ resetButtonStyle }>{ sprintf(
+								/* translators: %s: reset */
+								__( '%s' ), resetButtonText	) }
```

Comment:
__ and _x functions are not working with variables, you need to pass static text in them.

Also do we need sprintf here?

Reply:
Fixed